### PR TITLE
Replace Scal.io with Flare 

### PR DESCRIPTION
**Pull request by mnarinsky**

### CODE REVIEW

File: experts.md [hunk 1]
```diff
@@ -41,20 +41,12 @@ title: Bazel Community Experts
...
-          Scalio proliferates the adoption of Bazel, by bringing
-          digital transformation and advanced build pipeline tooling to
-          enterprise customers.  Their primary focus is in advancing
-          build/test tooling for all languages and platforms, working
-          with the world's top companies to move their organizations to
-          Bazel. Scalio's goal is in helping these companies to build
-          and test software at scale -- quickly and reliably. Find more
-          about Scalio’s Bazel initiatives
-          <a href="https://github.com/scalio/bazel-status" target="_blank">here on Github</a>.
+           Flare Build Systems, Inc. is the world’s first Bazel-focused consulting company. We’ve made a name for ourselves by assisting enterprises and high-growth, high-value startups migrate their build & test systems into Bazel, interacting with almost every major language and ruleset along the way, but we didn’t stop there - we also have launched a build, CI & engineering productivity-focused product line: flare.build.
```
Comment (jin) [inline experts.md]: linkify or remove?

**Comment by laurentlb** [comment]
Did Scal.io ask to be removed?

**Comment by mnarinsky** [comment]
> Did Scal.io ask to be removed?

Yes. Scal.io spun off a new company to focus on Bazel and RE API work

File: experts.md (hunk 1 above)
Comment (mnarinsky) [reply experts.md]: it's just a name of their product line

**Pull request merged**